packages/block-editor/src/components/block-tools/zoom-out-mode-inserters.js

```diff
@@ -64,41 +58,32 @@ function ZoomOutModeInserters() {
 		return null;
 	}
 
-	return [ undefined, ...blockOrder ].map( ( clientId, index ) => {
-		const shouldRenderInsertionPoint =
-			blockInsertionPointVisible && insertionPoint?.index === index;
+	const previousClientId = selectedBlockClientId;
+	const index = blockOrder.findIndex(
+		( clientId ) => selectedBlockClientId === clientId
+	);
+	const nextClientId = blockOrder[ index + 1 ];
 
-		const previousClientId = clientId;
-		const nextClientId = blockOrder[ index ];
-
-		const isSelected =
-			selectedBlockClientId === previousClientId ||
-			selectedBlockClientId === nextClientId;
-
-		return (
-			<BlockPopoverInbetween
-				key={ index }
-				previousClientId={ previousClientId }
-				nextClientId={ nextClientId }
-			>
-				{ ! shouldRenderInsertionPoint && isSelected && (
-					<ZoomOutModeInserterButton
-						onClick={ () => {
-							setInserterIsOpened( {
-								rootClientId: sectionRootClientId,
-								insertionIndex: index,
-								tab: 'patterns',
-								category: 'all',
-							} );
-							showInsertionPoint( sectionRootClientId, index, {
-								operation: 'insert',
-							} );
-						} }
-					/>
-				) }
-			</BlockPopoverInbetween>
-		);
-	} );
+	return (
+		<BlockPopoverInbetween
+			previousClientId={ previousClientId }
+			nextClientId={ nextClientId }
+		>
+			<ZoomOutModeInserterButton
```

Comment:
We lost the previous inserter. This only shows the inserter after the selected block. There should be one before and after the selected block.

Reply:
Yes, it interfered with the block toolbar.

Reply:
It will still interfere with the toolbar when you scroll down though, right? Is there a solution we can have without removing it entirely? The inserters don't render until they're present on the screen, so removing the top inserter highlights an issue where there is no focusable Add pattern button available for the keyboard on patterns that extend past the bottom of the viewport.

Reply:
> The inserters don't render until they're present on the screen, so removing the top inserter highlights an issue where there is no focusable Add pattern button available for the keyboard on patterns that extend past the bottom of the viewport.

@jeryj Would you be able to raise an Issue for that one? Any suggestions on a fix would be good. I assume it's "add back the inserters" but that will require input from Design so feel free to ping the design group here.

Reply:
https://github.com/WordPress/gutenberg/issues/66346